packages/cli/src/commands/start.ts

```diff
@@ -0,0 +1,33 @@
+import {spawn} from 'child_process'
+import {Command, flags} from '@oclif/command'
+
+export default class Start extends Command {
+  static description = 'Start development server'
+  static aliases = ['s']
+
+  static flags = {
+    help: flags.help({char: 'h'}),
+    production: flags.boolean({
+      char: 'p',
+      description: 'runs a production build',
+      default: false,
+      allowNo: false,
+    })
+  }
+
+  async run() {
+    const {flags} = this.parse(Start)
+    if (flags.production) {
+      var cp = spawn('next', ['build'], {stdio: 'inherit'})
```

Comment:
Don't think it matters right now really and don't think it is worth talking about here but I would vote for some kind of linting rule to limit `var` use in the future.